app/src/main/res/layout/activity_sign_up.xml

```diff
@@ -131,69 +149,79 @@
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
-                    android:layout_marginTop="20dp"
+                    android:layout_marginTop="16dp"
                     android:text="@string/password"
-                    android:textColor="#323842"
-                    android:textSize="15sp"
+                    android:fontFamily="@font/epilogue_medium"
+                    android:textColor="@color/light_black"
+                    android:textSize="20sp"
                     app:layout_constraintStart_toStartOf="@+id/tvEmail"
                     app:layout_constraintTop_toBottomOf="@+id/etEmail" />
 
                 <EditText
                     android:id="@+id/etPassword"
                     android:layout_width="match_parent"
                     android:layout_height="wrap_content"
+                    android:layout_marginHorizontal="36dp"
                     android:layout_marginTop="12dp"
+                    android:autofillHints="password"
+                    android:textSize="20sp"
                     android:background="@drawable/custom_edittext"
-                    android:elevation="5dp"
+                    android:elevation="4dp"
                     android:hint="@string/condition_password"
-                    android:padding="5dp"
                     android:inputType="textPassword"
-                    android:layout_marginHorizontal="50dp"
+                    android:fontFamily="@font/inter_regular"
+                    android:padding="4dp"
                     android:textColor="@color/black"
                     app:layout_constraintEnd_toEndOf="parent"
                     app:layout_constraintStart_toStartOf="parent"
                     app:layout_constraintTop_toBottomOf="@+id/tvPassword" />
 
                 <Button
                     android:id="@+id/btnSignUp"
-                    android:layout_width="193dp"
+                    android:layout_width="192dp"
                     android:layout_height="wrap_content"
-                    android:layout_marginTop="28dp"
-                    android:backgroundTint="#4DABF5"
+                    android:layout_marginTop="16dp"
+                    android:backgroundTint="@color/btn_color"
+                    android:fontFamily="@font/lato_bold"
                     android:text="@string/sign_up"
                     android:textColor="@color/white"
-                    android:textSize="20sp"
+                    android:textSize="24sp"
+                    app:layout_constraintBottom_toBottomOf="parent"
                     app:layout_constraintEnd_toEndOf="parent"
                     app:layout_constraintStart_toStartOf="parent"
-                    app:layout_constraintTop_toBottomOf="@+id/etPassword" />
+                    app:layout_constraintTop_toBottomOf="@+id/etPassword"
+                    app:layout_constraintVertical_bias="0.651" />
 
                 <TextView
```

Comment:
Wrap these text views in a Linear Layout and constrain the Linear Layout in the Constraint Layout accordingly.